server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,35 @@
+package psql
+
+import (
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+
+func (db *DB) AllMaintenanceRequests() ([]datastore.MaintenanceRequest, error) {
+	var maintenanceRequestTable []datastore.MaintenanceRequest
+	rows, err := db.connection.Query("SELECT * FROM maintenance_requests LIMIT $1", 777) //Number limits # of requests, can be anything
+	if err != nil {
+		return maintenanceRequestTable, err
```

Comment:
if there's an error we don't want to return anything for the MaintenanceRequest array, so you would instead do `return nil, err` This is true for all the other error return statements